### DISCUSSION TITLE
Can I run oobabooga text-generation-webui with 13B LLM model on a 4GB GPU + 16GB Ram?

### DISCUSSION THREAD

**Discussion by RuddroKRM1** [oobabooga/text-generation-webui#1880] [Q&A]
Hello. I'm trying to run oobabooga text-generation-webui with a 13B LLM model. I am getting this error.
![Capture](https://user-images.githubusercontent.com/132821220/236678679-484d8ef4-dadd-4ef5-a4bd-8893912f0095.PNG)
Is there any solution for it for my 4GB GTX970 GPU? I am quite new to this btw

**Comment by m-spangenberg** [comment]
Unfortunately your 4GB card won't be able to run [gpt4-x-alpaca-13b-native-4bit-128g](https://huggingface.co/anon8231489123/gpt4-x-alpaca-13b-native-4bit-128g). Have you tried running it in CPU mode? It will be slower, but it's better than nothing. 

People struggle getting [Pygmalion 6B to run on 6GB cards](https://www.reddit.com/r/PygmalionAI/comments/12dbfpk/cuda_out_of_memory_for_6_gb_vram_ooba/), so a 13B model would need something like 10 to 12GB, I'm guessing. 

I'm currently running [wizard-vicuna-13B-GGML](https://huggingface.co/TheBloke/wizard-vicuna-13B-GGML) on CPU with 16GB of RAM, and getting about 1.4 tokens/second. So all hope isn't lost. Good luck!
